cms/static/sass/elements/_header.scss

```diff
@@ -74,6 +74,7 @@
   .nav-dd {
 
     .title {
+      @extend %t-demi-strong;
```

Comment:
I've contained this work in separate commit that updates the markup for the header controls. This style goes with that and ensures a consistent look. I'm hesitant to make it its own PR though, because it would need to be merged at the same time as this one else the Help link in the header will look at bit off. What do you advise?